packages/core/test/scxml.test.ts

```diff
@@ -168,30 +168,30 @@ const testGroups = {
     // 'test210.txml', // sendidexpr not supported yet
     // 'test215.txml', // <invoke typeexpr="...">
     // 'test216.txml', // <invoke srcexpr="...">
-    // 'test220.txml', // done.invoke used as inexact event descriptor
+    'test220.txml',
     // 'test223.txml', // idlocation not implemented yet
-    // 'test224.txml', // done.invoke used as inexact event descriptor
+    // 'test224.txml', // <invoke idlocation="...">
     // 'test225.txml', // unique invokeids generated at invoke time
     // 'test226.txml', // <invoke src="...">
-    // 'test228.txml', // done.invoke used as inexact event descriptor + _event.invokeid not implemented yet
+    'test228.txml',
     'test229.txml',
-    // 'test230.txml', // done.invoke used as inexact event descriptor + SCXML _event properties not implemented yet
-    // 'test232.txml', // done.invoke used as inexact event descriptor
+    'test230.txml',
+    // 'test232.txml', // TODO: not sure
     // 'test233.txml', // <finalize> not implemented yet
     // 'test234.txml', // <finalize> not implemented yet
     'test235.txml',
-    // 'test236.txml', // done.invoke used as inexact event descriptor
-    // 'test237.txml', // done.invoke used as inexact event descriptor
-    // 'test239.txml', // done.invoke used as inexact event descriptor
+    // 'test236.txml', // TODO: not sure
```

Comment:
this block: 
https://github.com/davidkpiano/xstate/blob/8d328e92f74d0a428aa66a2395f9aa85bbba3223/packages/core/src/interpreter.ts#L219
should contain a similar logic as this:
https://github.com/davidkpiano/xstate/blob/8d328e92f74d0a428aa66a2395f9aa85bbba3223/packages/core/src/interpreter.ts#L426-L430
```suggestion
    // 'test236.txml', // reaching a final state should execute all onexit handlers
```